src/toil/lib/docker.py

```diff
@@ -229,7 +230,8 @@ def toil_job(job):
                                         user=user,
                                         environment=environment,
                                         **kwargs)
-
+            if sys.version_info >= (3, 0):
+                out = out.decode('utf-8')
```

Comment:
I don't think it's a good idea to do this here. What if someone is capturing binary data on stdout that is not valid UTF-8?

Reply:
Python 2 works differently so that the output works with "normal" strings which is why the tests failed.  You're right though, we should probably model the behavior off of the interior functions and change the tests instead.  Will change accordingly.

Reply:
Fixed.